docs/patterns/scheduler-agent-supervisor-content.md

```diff
@@ -53,7 +53,7 @@ You should consider the following points when deciding how to implement this pat
 
 - This pattern can be difficult to implement and requires thorough testing of each possible failure mode of the system.
 
-- The recovery/retry logic implemented by the Scheduler is complex and dependent on state information held in the state store. It might also be necessary to record the information required to implement a compensating transaction in a durable data store.
+- The recovery/retry logic implemented by the Scheduler is complex and dependent on state information held in the state store. It might also be necessary to record the information required to implement a compensating transaction in a durable data store. Note that compensating transactions don't always work.
```

Comment:
```suggestion
- The recovery/retry logic implemented by the Scheduler is complex and dependent on state information held in the state store. It might also be necessary to record the information required to implement a compensating transaction in a durable data store. A compensating transaction might fail as well.
```